views/cypress/tests/properties/simple.spec.js

```diff
@@ -123,20 +124,32 @@
             options.propertyType = propertiesInfo.singleSearchList.type;
             cy.addPropertyToClass(options);
         });
+
+        it('can edit and add new "Calendar" type property to the item class', function () {
```

Comment:
I have 2 questions: 

1) I saw we added only Calendar and Password, what about the other properties?
2) Is it ok to test it only in tao-item extension, but properties are part of many other extensions and mainly created through tao-core/generis extension?